docs/source/driver/postgresql.rst

```diff
@@ -19,7 +19,7 @@
 PostgreSQL Driver
 =================
 
-**Available for:** C/C++, GLib/Ruby, Go, Python, R
+.. adbc_driver_status:: ../../../c/driver/postgresql/README.md
```

Comment:
(commenting here because my change is outside the diff)

Should we remove the note below about "The PostgreSQL driver is in beta."? It appears to be out of date and we probably only want to track status in one place.